### PR TITLE
Update raspberry_pi_installation.md

### PR DESCRIPTION
**Pull request by MatsA**
Updated documentation and some pictures to better reflect actual version

### CODE REVIEW

File: raspberry_pi_installation.md [hunk 1]
```diff
@@ -70,17 +70,10 @@ Now you need to start the Node Server. Type...
 You should see the terminal output "signalk-server running at 0.0.0.0:3000" as shown below...
 
     $ signalk-server --sample-nmea0183-data
-    Using default configuration path: /home/pi/.signalk
-    Settings file does not exist, using empty settings
     Using sample data from /usr/lib/node_modules/signalk-server/samples/plaka.log
-    *** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
```
Comment (tkurki) [inline raspberry_pi_installation.md:76]: Have the Avahi warning disappeared? That would be indeed nice, they look pretty scary for somebody starting out.

File: raspberry_pi_installation.md [hunk 2]
```diff
@@ -177,30 +170,35 @@ and it will show something like
...
 If You have a NMEA0183 network, plug in Your NMEA0183 to USB interface and do the same procedure as for the Actisense above.
 With a configuration maybe looking like this
 
-![provider_nmea0183](https://user-images.githubusercontent.com/16189982/36218419-93f81c84-11b4-11e8-86c2-8319b9013b90.jpeg)
+![SK_N1](https://user-images.githubusercontent.com/16189982/69761986-44b25e80-1169-11ea-9401-2c06ace9eb3b.png)
 
 ## Sample files
 
-If You don’t have any NMEA interface hardware You could set up a provider with the sample files instead.
+If You don’t have any NMEA interface hardware You could set up a "File Stream" connection with the sample files instead.
```
Comment (tkurki) [inline raspberry_pi_installation.md:191]: We already have --sample-nmea0183-data documented here, maybe we could just remove this Sample files section?

File: raspberry_pi_installation.md (hunk 1 above)
Comment (MatsA) [reply raspberry_pi_installation.md:76]: Jepp, did a fresh install for some time ago and now running 1.18 w/o any warnings

File: raspberry_pi_installation.md (hunk 2 above)
Comment (MatsA) [reply raspberry_pi_installation.md:191]: Njaa......
Earlier in the text You are advised to use `signalk-server --sample-nmea0183-data`, to check the installation, that's true, but after You have used the `sudo signalk-server-setup` it's not that easy for a newbie to run the test again. This is due to that You have to stop the 2 deamons before using it again, and then, after the test, restart. Also if You want to use the N2K data instead, to test `canboat`, there is no info available. Another thought is that this info also could help OpenPlottter users finding test files.

**Pull request merged**